packages/wxt/src/core/utils/building/internal-build.ts

```diff
@@ -28,8 +28,11 @@ import { isCI } from 'ci-info';
  * 2. Executes the rebuild function with a blank previous output so everything is built (see
  *    `rebuild` for more details)
  * 3. Prints the summary
+ * @param {string} [allowAllEntryPoints] - If true, all entrypoints are returned.
  */
-export async function internalBuild(): Promise<BuildOutput> {
+export async function internalBuild(
+  allowAllEntryPoints?: boolean,
+): Promise<BuildOutput> {
```

Comment:
Undo this whole file. Instead of modifying the existing build functions, you should just call `findEntrypoints` inside `zip.ts` before we actually generate the ZIP.

Probably before this line:
https://github.com/wxt-dev/wxt/blob/4ce178c592ec106955697a4c4bc32cc0f24da65e/packages/wxt/src/core/zip.ts#L58